addons/website_crm_livechat/models/mail_channel.py

```diff
@@ -17,3 +17,10 @@ def _convert_visitor_to_lead(self, partner, key):
             visitor_sudo.write({'lead_ids': [(4, lead.id)]})
             lead.country_id = lead.country_id or visitor_sudo.country_id
         return lead
+
+    def _chatbot_prepare_lead_values(self, step_id, description):
```

Comment:
Note that in commit about "adding chatbot" you have code in website_crm_livechat (isn't it related to the next commit ?)